### ISSUE TITLE
Some of our CIs are failing as `protoc` is not installed in the containers used to build the agent

### ISSUE THREAD

**Issue by fidencio** [kata-containers/kata-containers#3323]
We seem to start requiring `protoc` (@Jakob-Naucke, you remember which commit forced that dependency?) and we're failing to build the agent on CI jobs that are using the `alpine` image, which doesn't have `protoc` installed.

Let's install `protoc`, unblock our CI, and open a new issue for a deeper investigation on whether this is something we should add to all the containers used to build the rootfs or not.

**Comment by Jakob-Naucke** [comment]
> We seem to start requiring `protoc` and we're failing to build the agent on CI jobs that are using the `alpine` image, which doesn't have `protoc` installed.

Sorry, which ones? http://jenkins.katacontainers.io/job/kata-containers-2.0-vfio-PR/ seems sorta healthy.

> @Jakob-Naucke, you remember which commit forced that dependency?

I _thought_ it was the still open #3297 but apparently not 😄
